### PR TITLE
chore: release/2.6.3

### PR DESCRIPTION
**Pull request by Szymon-dziewonski**
# Related issue

Closes #

# Scope of work

Update dependencies inside packages
Update code connected to dependencies change

# Screenshots of visual changes

# Checklist

- [ ] Self code-reviewed
- [ ] Changes documented
- [ ] Semantic HTML
- [ ] SSR-friendly
- [ ] Caching friendly
- [ ] a11y for WCAG 2.0 AA
- [ ] examples created
- [ ] blocks created
- [ ] cypress tests created

### CODE REVIEW

**Review by dawid-ziobro — APPROVED** [review]

File: apps/test/vue/cypress.config.ts [hunk 1]
```diff
@@ -21,7 +22,7 @@ export default defineConfig({
     experimentalSingleTabRunMode: true,
     setupNodeEvents(on, config) {
       initPlugin(on, config);
-      isCoverageEnabled && require('@cypress/code-coverage/task')(on, config);
+      isCoverageEnabled && coverageTask(on, config);
```
Comment (FRSgit) [inline apps/test/vue/cypress.config.ts:25]: code deoptimization😄

File: packages/sfui/frameworks/react/hooks/usePopover/types.ts [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+import type { Middleware } from '@floating-ui/react-dom';
 import { SfPopoverPlacement, SfPopoverStrategy, type SfPopoverBasePlacement } from '@storefront-ui/shared';
 
 export { SfPopoverPlacement, SfPopoverStrategy, type SfPopoverBasePlacement };
 
 export interface UsePopoverOptions {
   isOpen?: boolean;
   placement?: `${SfPopoverPlacement}`;
-  middleware?: UseFloatingProps['middleware'];
+  middleware?: Array<Middleware | null | undefined | false>;
```
Comment (FRSgit) [inline packages/sfui/frameworks/react/hooks/usePopover/types.ts:9]: Question(non-blocking): why `null | undefined | false`?

File: packages/sfui/frameworks/react/hooks/usePopover/usePopover.ts [hunk 3]
```diff
@@ -19,8 +19,8 @@ export function usePopover(options: UsePopoverOptions) {
 
   const style = {
     position: strategy,
-    top: y ?? 0,
-    left: x ?? 0,
+    top: y,
+    left: x,
```
Comment (FRSgit) [inline packages/sfui/frameworks/react/hooks/usePopover/usePopover.ts:23]: Question: default values are not needed anymore?

**Review by FRSgit — APPROVED** [review]

File: apps/test/vue/cypress.config.ts (hunk 1 above)
Comment (Szymon-dziewonski) [reply apps/test/vue/cypress.config.ts:25]: i cannot use now `require` in es module

File: packages/sfui/frameworks/react/hooks/usePopover/usePopover.ts (hunk 3 above)
Comment (Szymon-dziewonski) [reply packages/sfui/frameworks/react/hooks/usePopover/usePopover.ts:23]: now x,y return always number not like previously `null` https://github.com/floating-ui/floating-ui/releases/tag/%40floating-ui%2Freact-dom%402.0.0

File: apps/test/vue/cypress.config.ts (hunk 1 above)
Comment (FRSgit) [reply apps/test/vue/cypress.config.ts:25]: yup, I know 😭

**Pull request merged**